api/src/owner_rpc.rs

```diff
@@ -1338,7 +1340,8 @@ pub fn run_doctest_owner(
 ) -> Result<Option<serde_json::Value>, String> {
```

Comment:
this function has a lot of `unwrap`. I understand that it is a test helper but even so, it seems un-necessary not sure how difficult the refactor of this return type would be

Reply:
I think unwraps in tests are okay for now, happy to see a PR later that removes them (a lot of these tests were written before you could return a Result from main or from a test).